Sources/Sentry/SentryOptions.m

```diff
@@ -393,6 +393,12 @@ - (BOOL)isTracingEnabled
         || _tracesSampler != nil;
 }
 
+- (void)setDebug:(BOOL)debug
```

Comment:
m: Do you think is worth having another function with `setDebug:withLevel` or something like this?